notebooks/gammanu.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    help(lambdify((nu, t), gammanu, modules="scipy"))
</pre><p>Can we make it name it <code>F_nu_lambdifygenerated</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore-research/pyscf-ipu/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>not that I can easily see in the lambdify doc.</p>